src/webgpu/shader/validation/expression/call/builtin/const_override_validation.ts

```diff
@@ -147,23 +147,26 @@ export function stageSupportsType(stage: ConstantOrOverrideStage, type: Type) {
  * @param expectedResult false if an error is expected, true if no error is expected
  * @param args the arguments to pass to the builtin
  * @param stage the evaluation stage
+ * @param returnType the explicit return type of the result variable, if provided (implicit otherwise)
  */
 export function validateConstOrOverrideBuiltinEval(
   t: ShaderValidationTest,
   builtin: string,
   expectedResult: boolean,
   args: Value[],
-  stage: ConstantOrOverrideStage
+  stage: ConstantOrOverrideStage,
+  returnType?: Type
```

Comment:
Once this lands, please can you file a bug to fix up other tests to use this? It's a good thing to check.

Reply:
Yes: https://github.com/gpuweb/cts/issues/3613